CHANGELOG.md

```diff
@@ -0,0 +1,112 @@
+# CHANGELOG
+
+## tmp v0.2.0
+
+- drop support for node version < v8.17.0
```

Comment:
I've just realized that [Node v8 was EOL](https://nodejs.org/en/about/releases/) in December.

Reply:
is it ok to release it for v8 (as its working) and drop support in feature version?

Reply:
Yes, just wanted to note.